cpp/include/cudf/detail/nvtx/nvtx3.hpp

```diff
@@ -0,0 +1,1924 @@
+/*
+ *  Copyright 2019 Jacob Hemstad
```

Comment:
I think this should be an NVIDIA copyright. :)

Reply:
Ah yeah, missed that.